processes-services/src/main/java/org/exoplatform/processes/notification/provider/MailTemplateProvider.java

```diff
@@ -65,7 +69,7 @@ protected MessageInfo makeMessage(NotificationContext notificationContext) {
         String processTitle = notificationInfo.getValueOwnerParameter(NotificationArguments.REQUEST_PROCESS.getKey());
         templateContext.put("PROCESS_TITLE", encoder.encode(processTitle));
         templateContext.put("REQUEST_TITLE", encoder.encode(requestTitle));
-        templateContext.put("REQUEST_DESCRIPTION", encoder.encode(requestDescription));
+        templateContext.put("REQUEST_DESCRIPTION",encodeTextInsideHtmlTags(encoder, requestDescription));
```

Comment:
Let's just check if the text is sanitized with HTMLSanitizer , else no n eed to encode it .

Reply:
https://github.com/exoplatform/processes/blob/6567ba8d32d27a150750ad4eaa1cb75b9c390edc/processes-services/src/main/java/org/exoplatform/processes/rest/util/EntityBuilder.java#L213